vendor-extra/JatsContentBundle/tests/EventListener/BodyListenerTest.php

```diff
@@ -102,7 +102,7 @@ public function it_adds_the_body_content(string $xml, array $context, array $exp
         $this->assertEquals(
             array_map(
                 function (array $block) : TemplateView {
-                    return new TemplateView(null, $block);
+                    return new TemplateView('', $block);
```

Comment:
This is a bit of a pain; becoming clear that https://github.com/libero/browser/blob/f02935fc996e5df6d497ad206d658fbf710b8488/vendor-extra/LiberoPageBundle/tests/ViewConvertingTestCase.php#L17-L27 needs refactoring, but avoiding doing so here.